raiden/ui/console.py

```diff
@@ -153,7 +113,7 @@ def lasterr(n: int = 1) -> None:
         print("Entering Console" + OKGREEN)
         print("Tip:" + OKBLUE)
         print_usage()
-        IPython.start_ipython(argv=["--gui", "gevent"], user_ns=self.console_locals)
+        IPython.start_ipython(argv=[], user_ns=self.console_locals)
```

Comment:
nice! ipython now has native gevent support?

Reply:
No, but see above. Due to monkeypatching it seems to work automatically.